packages/preset-uno/src/rules/animation.ts

```diff
@@ -14,4 +15,37 @@ export const animations: Rule[] = [
     return `${keyframes[name]}\n${constructCSS({ animation: `${name} 1s linear infinite` })}`
   }],
   ['animate-none', { animation: 'none' }],
+  [/^animation(?:-duration)?-((\d+)(?:(s|ms)?))$/, ([, d]) => {
+    return {
+      'animation-duration': h.bracket.time(d.replace(/-duration/, '')),
+    }
+  }],
+  [/^animation-delay-((\d+)(?:(s|ms)?))$/, ([, d]) => {
+    return {
+      'animation-delay': h.bracket.time(d),
+    }
+  }],
+  [
+    /^animation-(?:fill-)?mode-(none|forwards|backwards|both)$/, ([, d]) => ({
+      'animation-fill-mode': d,
```

Comment:
what about global values https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode#syntax?

Reply:
I don't know if we want to handle global values, but the spec is there